index.html

```diff
@@ -221,6 +221,87 @@ <h1 class="post">Multi-Post Stories</h1>
           <button class="bt" type="button">see project</button>
         </div>
       </section>
+      <section class="contact">
+        <h1 class="co">CONTACT US</h1>
+        <p class="descrip">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form class="form" action="#" method="post">
+          <ul class="fo">
+            <li class="remo">
+              <input
+                class="te"
+                type="text"
+                id="name"
+                name="user"
```

Comment:
- [ ] Your form does not submit to Formspree because you have `action="#"`. Please replace that with the accurate action URL similar to this `https://formspree.io/f/{form_id}` 😎 
- [ ] I'm trying to understand your use of unordered lists here. In my opinion, you can either have your `<input>` as is or enclose them in a `<div>`